src/Microsoft.PowerShell.ConsoleHost/host/msh/ManagedEntrance.cs

```diff
@@ -31,9 +31,9 @@ public sealed class UnmanagedPSEntry
         public static int Start(string consoleFilePath, [MarshalAs(UnmanagedType.LPArray, ArraySubType = UnmanagedType.LPWStr, SizeParamIndex = 2)]string[] args, int argc)
 #pragma warning restore 1573
         {
+            System.Management.Automation.Runspaces.EarlyStartup.Init();
```

Comment:
Style. Please insert new line.
```suggestion
            System.Management.Automation.Runspaces.EarlyStartup.Init();

```